src/scanoss/api/common/v2/scanoss_common_pb2_grpc.py

```
@@ -1,4 +1,24 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 """Client and server classes corresponding to protobuf-defined services."""
 import grpc
+import warnings
```

Ruff (0.11.9)

`warnings` imported but unused

Remove unused import: `warnings`

(F401)